src/core/services/theming/theming.spec.js

```diff
@@ -758,7 +769,7 @@ describe('$mdTheming service', function() {
 describe('md-theme directive', function() {
   beforeEach(module('material.core'));
 
-  it('should observe and set mdTheme controller', inject(function($compile, $rootScope) {
+  it('should watch and set mdTheme controller', inject(function($compile, $rootScope) {
```

Comment:
Maybe move the inject part on a newline to help with readability and line length.

This is also inconsistent within this file.